src/libraries/sendtohelixhelp.proj

```diff
@@ -62,6 +62,17 @@
     <HelixType Condition="'$(BuildAllConfigurations)' == 'true'">test/functional/packaging/</HelixType>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(TargetOS)' == 'Browser'">
+    <IncludeDotNetCli>true</IncludeDotNetCli>
+    <DotNetCliPackageType>sdk</DotNetCliPackageType>
+    <DotNetCliVersion>3.1.201</DotNetCliVersion>
+  </PropertyGroup>
+  <ItemGroup Condition="'$(TargetOS)' == 'Browser'">
+    <HelixPreCommand Include="${HELIX_CORRELATION_PAYLOAD}/dotnet/dotnet tool install --tool-path ${HELIX_WORKITEM_PAYLOAD}/xharness-cli --version $(MicrosoftDotNetXHarnessCLIVersion) --add-source https://dnceng.pkgs.visualstudio.com/public/_packaging/dotnet-eng/nuget/v3/index.json Microsoft.DotNet.XHarness.CLI" />
+    <HelixPreCommand Include="export DOTNET_ROOT=${HELIX_CORRELATION_PAYLOAD}/dotnet" />
+    <HelixPreCommand Include="export PATH=${HELIX_WORKITEM_PAYLOAD}/xharness-cli:$PATH" />
```

Comment:
This will improve the XHarness logs:
```suggestion
    <HelixPreCommand Include="export PATH=${HELIX_WORKITEM_PAYLOAD}/xharness-cli:$PATH" />
    <HelixPreCommands>$(HelixPreCommands) export XHARNESS_DISABLE_COLORED_OUTPUT=true;</HelixPreCommands>
    <HelixPreCommands>$(HelixPreCommands) export XHARNESS_LOG_WITH_TIMESTAMPS=true;</HelixPreCommands>
```

However, it would be better to add these commands only for CI, not local runs. Not sure how to do that?

Reply:
Oh, sorry @directhex.. this should be the ItemGroup now. I forgot you changed that in runtime